ludwig/api.py

```diff
@@ -89,6 +90,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Set dask config to not use placement groups
+dask.config.set(annotations={"ray_remote_args": {"placement_group": None}})
```

Comment:
This should be done somewhere in the RayBackend or DaskEngine classes.

Reply:
Moved to DaskEngine's initialization.

Reply:
Actually, setting dask config from within either `RayBackend` or `DaskEngine` classes wasn't working. I've now moved this to `__main__` inside `backend/ray.py`.